csxtools/lmfit_models.py

```diff
@@ -0,0 +1,200 @@
+import numpy as np

```

Comment:
They were useful for fitting the 2D detector data. But I agree they are general enough and should rather be in scikit-beam.